test/src/test/java/org/corfudb/runtime/CorfuRuntimeTest.java

```diff
@@ -139,7 +138,7 @@ public void doesNotUpdateToLayoutWithSmallerEpoch() throws Exception {
 
         rt.getLayoutView().updateLayout(currentLayout, 0);
 
-        assertThat(getLayoutServer(SERVERS.PORT_2).getCurrentLayout().getEpoch()).isEqualTo(1);
+        assertThat(getLayoutServer(SERVERS.PORT_2).getCurrentLayout().getEpoch() == 1);
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Complete the assertion. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2970)